Mathlib/Tactic/ModCases.lean

```diff
@@ -74,13 +74,15 @@ partial def proveOnModCases {u : Level} (n : Q(ℕ)) (a : Q(ℤ)) (b : Q(ℕ)) (
     let (pr, acc) ← proveOnModCases n a b1 p
     pure (q(onModCases_succ $b $g $pr), g.mvarId! :: acc)
 
+#adaptation_note /-- kmill 2025-06-28 Added `unusedHavesSuffices` since `p₁` is "unused". -/
```

Comment:
Is there some mechanism for Qq to opt out of this, or mark the `=Q`  assumption as used?